regolith/builders/preslistbuilder.py

```diff
@@ -31,12 +31,31 @@ def construct_global_ctx(self):
         gtx['str'] = str
         gtx['zip'] = zip
 
+    def list_group_members(self, grp):
+        # get all group members
+        grpmembers = []
+        print(self.gtx['people'][0]['_id'])
+        for person in self.gtx['people']:
+            for position in person.get('education', {}):
+                if position.get('group', None) == grp:
+                    if person['_id'] not in grpmembers:
+                        grpmembers.append(person['_id'])
+            for position in person.get('employment', {}):
+                if position.get('group', None) == grp:
+                    if person['_id'] not in grpmembers:
+                        grpmembers.append(person['_id'])
+        return grpmembers
+
+
     def latex(self):
         """Render latex template"""
         for group in self.gtx['groups']:
             pi = fuzzy_retrieval(self.gtx['people'], ['aka', 'name', '_id'],
                                  group['pi_name'])
-
+        listgrpmembers = self.list_group_members('bg')
```

Comment:
Would it be possible to build preslists for every member of the group? Maybe name them `presentations_<id>.tex`?